Source/Blazorise/Components/Modal/ModalContent.razor.cs

```diff
@@ -75,6 +108,21 @@ internal void NotifyHasModalFooter()
         /// </summary>
         protected virtual bool AsDialog => false;
 
+        /// <summary>
+        /// True if modal should keep the input focus.
+        /// </summary>
+        protected bool ActivateFocusTrap => Options?.ActivateModalFocusTrap == true;
```

Comment:
I'm sure someone will eventually want to configure this per modal, can we have it as a Parameter also?

Reply:
Yep

Reply:
Added